tests/unittests/config/test_cc_ubuntu_advantage.py

```diff
@@ -197,160 +210,218 @@ def test_schema_validation(self, config, error_msg):
                 validate_cloudconfig_schema(config, get_schema(), strict=True)
 
 
-class TestHandle(CiTestCase):
-
-    with_logs = True
-
-    def setUp(self):
-        super(TestHandle, self).setUp()
-        self.tmp = self.tmp_dir()
+class TestHandle:
 
-    @mock.patch("%s.maybe_install_ua_tools" % MPATH)
-    def test_handle_no_config(self, m_maybe_install_ua_tools):
-        """When no ua-related configuration is provided, nothing happens."""
-        cfg = {}
-        handle("ua-test", cfg=cfg, cloud=None, log=self.logger, args=None)
-        self.assertIn(
-            "DEBUG: Skipping module named ua-test, no 'ubuntu_advantage'"
-            " configuration found",
-            self.logs.getvalue(),
-        )
-        self.assertEqual(m_maybe_install_ua_tools.call_count, 0)
-
-    @mock.patch("%s.configure_ua" % MPATH)
-    @mock.patch("%s.maybe_install_ua_tools" % MPATH)
-    def test_handle_tries_to_install_ubuntu_advantage_tools(
-        self, m_install, m_cfg
-    ):
-        """If ubuntu_advantage is provided, try installing ua-tools package."""
-        cfg = {"ubuntu_advantage": {"token": "valid"}}
-        mycloud = FakeCloud(None)
-        handle("nomatter", cfg=cfg, cloud=mycloud, log=self.logger, args=None)
-        m_install.assert_called_once_with(mycloud)
+    mycloud = FakeCloud(None)
```

Comment:
We should probably also migrate to [tests/unittests/util.py:get_cloud](https://github.com/canonical/cloud-init/blob/main/tests/unittests/util.py#L7) where possible instead of FakeCloud. The goal is to drop all test-module-local FakeCloud definitions from each unit test module when possible.

Reply:
Done.